### PR TITLE
add prefix of aif to input types

### PR DESCRIPTION
**Pull request by ninjanails**
@wytrych Please review :)

### CODE REVIEW

**Comment by wytrych** [comment]
Nice! You need to update the Changelog too. This is a breaking change.

**Comment by ninjanails** [comment]
Okay ill do that now thanks :)

**Comment by ninjanails** [comment]
Update the change log is it OK?

File: bower.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "angular": "1.4",
     "venturocket-angular-slider": "~0.3",
     "angular-bind-html-compile": "~1.1",
-    "angular-gettext": "0.2.8"
+    "angular-gettext": "0.2.8",
+    "angular-input-fields": "~5.0.0"
```
Comment (wytrych) [inline bower.json]: This shouldn't be here, don't know how it happened

Comment (ninjanails) [inline bower.json]: So I just need to remove `"angular-input-fields": "~5.0.0"` ?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 6.0.0
+
+* BREAKING CHANGE: Add prefix of aif to input types
```
Comment (Minglee01) [inline CHANGELOG]: Id probably say  `of aif to all the directives and updated templates and tests`

Comment (Minglee01) [inline CHANGELOG]: just ignore this.

**Pull request merged**